glusterd2/brick/validation.go

```diff
@@ -19,12 +19,53 @@ const (
 )
 
 // InitChecks is a set of checks to be run on a brick
+//if bool value is  ture --> dont do any check
 type InitChecks struct {
-	IsInUse  bool
-	IsMount  bool
-	IsOnRoot bool
+	IsInUse          bool
+	IsMount          bool
+	IsOnRoot         bool
+	IsCreateBrickDir bool
 }
 
+// NewInitCheck initializes and returns InitCheck struct
+func NewInitCheck() *InitChecks {
+	return &InitChecks{
+		IsInUse:          true,
+		IsMount:          true,
+		IsOnRoot:         true,
+		IsCreateBrickDir: true,
+	}
+}
+
+// PrepareChecks initializes InitChecks based on req
+func (c *InitChecks) PrepareChecks(force bool, req map[string]bool) *InitChecks {
+
+	//if force == true disable all checks
+	if force {
```

Comment:
You don't have to explicitly do this as `false` is default value for bool. See comment below about merging  `NewInitCheck()` and `InitChecks()`

Reply:
done